tools/tasks/install.ts

```diff
@@ -85,27 +85,33 @@ export default {
           const dir = await $.workingDir.join(".metatype").ensureDir();
           return await $.co(
             ["command", "reactor", "proxy"].map((kind) => {
-              const url = `https://github.com/bytecodealliance/wasmtime` +
+              const url =
+                `https://github.com/bytecodealliance/wasmtime` +
                 `/releases/download/v${WASMTIME_VERSION}/wasi_snapshot_preview1.${kind}.wasm`;
-              const token = Deno.env.get("GITHUB_TOKEN") ||
-                Deno.env.get("GH_TOKEN");
+              const token =
+                Deno.env.get("GITHUB_TOKEN") || Deno.env.get("GH_TOKEN");
               const headers = token
                 ? {
-                  "Authorization": `Bearer ${token}`,
-                }
+                    Authorization: `Bearer ${token}`,
+                  }
                 : {};
-              return $.request(url).header(headers)
+              return $.request(url)
+                .header(headers)
                 .showProgress()
-                .pipeToPath(
-                  dir.join(std_url.basename(url)),
-                  {
-                    create: true,
-                  },
-                );
-            }),
+                .pipeToPath(dir.join(std_url.basename(url)), {
+                  create: true,
+                });
+            })
           );
         },
       });
     },
   },
+  // "install-protobuf-compiler": {
+  //   fn: ($) => $`sudo apt install protobuf-compiler`,
+  // },
+  // Enable rust codegen for protoc
+  "install-protobuf-rust": {
+    fn: ($) => $`cargo install protobuf-codegen@3.5.1`,
```

Comment:
You can use cargo binstall for this. Or better yet, use a `cargobi` port.

Reply:
Thanks, I'll keep this in mind when tackling other stuff.